docs/fundamentals/documentarray/serialization.md

```diff
@@ -358,43 +357,3 @@ To build a DocumentArray from dataframe,
 df = ...
 da = DocumentArray.from_dataframe(df)
 ```
-
-## From/to cloud
```

Comment:
I think removing this section is breaking for some of our content
In this PR I chose to keep it and document cloud support further in a separate section